pyproject.toml

```diff
@@ -82,7 +82,7 @@ extra-dependencies = [
   "mcp ; python_version >= '3.10'",
 ]
 
-[[tool.hatch.envs.all.matrix]]
+[[tool.hatch.envs.test.matrix]]
```

Comment:
unrelated: I think that the test matrix is configured in the GitHub workflow
https://github.com/deepset-ai/hayhooks/blob/e4c2a55fabb12f9fd114e1bb055de94d52972ba8/.github/workflows/tests.yml#L23
so`[[tool.hatch.envs.test.matrix]]` can be removed from pyproject for clarity

Reply:
Yeah, I've removed it otherwise each matrix run will run a local matrix run and this is not ideal ;)

But actually, I wanted a faster way to iterate on tests dev on specific python versions locally. But for this, I may open another PR and duplicate the `test` environment (e.g. `test-local-matrix`) adding matrix only on this one. WDYT?

Reply:
Ah, no, sorry. Feel free to leave it as it is. (Maybe just add a comment about this point.)

Reply:
No worries, I'll open another PR for that (and maybe document the why) ;)